usecase/network/main.go

```diff
@@ -147,85 +147,85 @@ func (m *Main) Setup() error {
 }
 
 func (m *Main) extractAndPost(packets chan gopacket.Packet) {
-	var profileID uint64
+	var columnID uint64
```

Comment:
I'm gonna stick with row/col if that's ok, because I want people to consider the Pilosa data model when they're reading the code. I'm not opposed to changing it if you feel strongly that it would be more readable though.